src/js/errorMiddleware.js

```diff
@@ -0,0 +1,63 @@
+/*
+
+    A default error handler that relays an upstream error to the user. See the documentation for details:
+
+    https://github.com/GPII/gpii-express/blob/master/docs/errorMiddleware.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.express.middleware.error");
+
+/**
+ *
+ * @param that - The middleware component itself.
+ * @param error - The raw upstream error.
+ * @param request {Object} The Express {Request} object.
+ * @param response {Object} The Express {Response} object.
+ * @param next {Function} The next piece of middleware in the chain.  Only used if a response has already been sent (i.e. we have no way of contacting the user).
+ *
+ */
+gpii.express.middleware.error.sendError = function (that, error, request, response, next) {
+    if (response.headersSent) {
+        return next(error);
```

Comment:
This is a little odd - firstly, who reads this return value - secondly, why don't we communicate an error specific to the situation rather than punting the original error - thirdly - if the headers have been sent, isn't it likely that any further attempt to communicate the error via the request will fail? This looks like a "log a high priority warning or terminate" situation to me. A few stray blank lines below